openid-4-verifiable-presentations-1_0.md

```diff
@@ -2001,68 +2028,263 @@ Note: The `nonce` and `aud` are set to the `nonce` of the request and the Client
 
 # IANA Considerations
 
-## Response Types
+## OAuth Authorization Endpoint Response Types Registry
+
+This specification registers the following `response_type` values
+in the IANA "OAuth Authorization Endpoint Response Types" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6749].
+
+### vp_token
 
 * Response Type Name: `vp_token`
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
-* Specification Document(s): https://openid.net/specs/openid-4-verifiable-presentations-1_0.html
+* Specification Document(s): (#response) of this specification
+
+### vp_token id_token
 
 * Response Type Name: `vp_token id_token`
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
-* Specification Document(s): https://openid.net/specs/openid-4-verifiable-presentations-1_0.html
+* Specification Document(s): (#response) of this specification
 
-Note: Plan to register the following Response Types in the [OAuth Authorization Endpoint Response Types IANA Registry](https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#endpoint).
+## OAuth Parameters Registry
 
-## Media Types
-### application/verifier-attestation+jwt {#va_media_type}
+This specification registers the following OAuth parameters
+in the IANA "OAuth Parameters" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6749].
+
+### presentation_definition
+
+* Name: presentation_definition
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### presentation_definition_uri
+
+* Name: presentation_definition_uri
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### client_metadata
+
+* Name: client_metadata
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### request_uri_method
+
+* Name: request_uri_method
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#vp_token_request) of this specification
+
+### wallet_nonce
+
+* Name: wallet_nonce
+* Parameter Usage Location: authorization request
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#request_uri_method_post) of this specification
+
+### vp_token
+
+* Name: vp_token
+* Parameter Usage Location: authorization response
```

Comment:
Technically, can also be token response. That is why OpenID4VP spec uses a term `response` and not `authorization response`, because the flow is not limited to one request-response flow, but could also be used in authorization code flow.

same for `presentation_submission` parameter below.

```suggestion
* Parameter Usage Location: authorization response or token response
```